app/templates/visualization.hbs

```diff
@@ -29,6 +29,7 @@
       {{component visualizationComponent
         classNames='buildermod__viz'
         immutableData=immutableData
+        modelData=modelData
```

Comment:
`immutableData` are filtered data and is the original data being used. I've added `modelData` to figure out some global dataset attributes properties (like axis domains) on the fly by the viz component. Those properties could have been found by the controller but would have been very attribute specific (e.g. min/max for distance, complexity, ..).